DiamondLightSource/ispyb-database
# Remove upsert xray centring proc

Given that the table this procedure references is now removed the procedure should be too.

To test:
* Confirm removed procedures reference old tables that have also been removed
* Confirm no other errant places referencing the old table
